modules/installation-guide/partials/proc_configuring-communication-between-che-components.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-communication-between-{prod-id-short}-components_{context}"]
+= Configuring communication between {prod-short} components
+
+To communicate together, {prod-short} components use the internal services names, which are exposed in the internal {platforms-name} network.
+
+The administration can disable the use of the internal services names to force {prod-short} components to use external {platforms-ingress}. Using the external {platforms-ingress} might slow the traffic and lead to issues because it uses proxies, certificates, and firewalls.
+
+Using the external {platforms-ingress} is necessary in the following situations:
+
+* To deploy {prod-short} across multiple {platforms-name} clusters. The internal services names are not shared between distinct clusters.
+* To deploy {prod-short} across multiple namespaces. The {orch-name} NetworkPolicies restricts communication between namespaces.
+* To deploy {prod-short} with the multitenant network plugin.
+
```

Comment:
```suggestion
[IMPORTANT]
====
Using the external {platforms-ingress} might slow the traffic and lead to issues because it uses proxies, certificates, and firewalls.
====

```